src/app/views/directory.php

```diff
@@ -44,8 +49,8 @@
             </article>
         </div>
         <script>
-        var links = document.querySelectorAll('.item-list a');
-        for (i = 0; i < links.length; ++i) {
+	var links = document.querySelectorAll('.item-list a');
```

Comment:
Were these 2 lines supposed to be changed in this PR? (Again indentation is changed, that's all.)